frameit/lib/frameit/device_types.rb

```diff
@@ -4,6 +4,11 @@ module Color
     SILVER = "Silver"
     ROSE_GOLD = "Rose Gold"
     GOLD = "Gold"
+    RED = "Red"
+    YELLOW = "Yellow"
+    WHITE = "White"
+    CORAL = "Coral"
+    BLUE = "Blue"
```

Comment:
I _don't_ think we can actually add these colors right now. We do not have frames with these colors so this will only cause confusion and break things right now 😔 

For reference: We get our frames from https://facebook.design/devices so we can add more colors if/when these frames are updated. Until then, we can not support more colors 😬